Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -262,18 +276,18 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         }
 
         switch (propertyName) {
-        case "font-size":
-            return new WI.DetailsSectionSimpleRow(WI.UIString("Size", "Size @ Font Details Sidebar Property", "Property title for `font-size`."));
-            break;
-        case "font-style":
-            return new WI.DetailsSectionSimpleRow(WI.UIString("Style", "Style @ Font Details Sidebar Property", "Property title for `font-style`."));
-            break;
-        case "font-weight":
-            return new WI.DetailsSectionSimpleRow(WI.UIString("Weight", "Weight @ Font Details Sidebar Property", "Property title for `font-weight` and `wght` variation axis."));
-            break;
-        case "font-stretch":
-            return new WI.DetailsSectionSimpleRow(WI.UIString("Stretch", "Stretch @ Font Details Sidebar Property", "Property title for `font-stretch`."));
-            break;
+            case "font-size":
+                return new WI.DetailsSectionSimpleRow(WI.UIString("Size", "Size @ Font Details Sidebar Property", "Property title for `font-size`."));
+                break;
+            case "font-style":
+                return new WI.DetailsSectionSimpleRow(WI.UIString("Style", "Style @ Font Details Sidebar Property", "Property title for `font-style`."));
+                break;
+            case "font-weight":
+                return new WI.DetailsSectionSimpleRow(WI.UIString("Weight", "Weight @ Font Details Sidebar Property", "Property title for `font-weight` and `wght` variation axis."));
+                break;
+            case "font-stretch":
+                return new WI.DetailsSectionSimpleRow(WI.UIString("Stretch", "Stretch @ Font Details Sidebar Property", "Property title for `font-stretch`."));
+                break;
```

Comment:
Please don't update the spacing of unchanged code. It makes it difficult over time to track in which commit a line was added or significantly changed when running something like `git blame`.

Ditto for other places in this PR.